cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -69,11 +69,13 @@ spec:
               resourceFieldRef:
                 containerName: {{ .Chart.Name }}-init
                 resource: limits.cpu
+                divisor: "0"
```

Comment:
Shouldn't we set to its default, which is documented as "1" [here](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#store-container-fields)? 

> The divisor field is optional and has the default value of 1. A divisor of 1 means cores for cpu resources, or bytes for memory resources.

Reply:
Yeah, that was my understanding too, I found a lot of examples setting it to "0", but couldn't really get why's that. I'll dig into it a bit more.

Reply:
The current chart results in a default of "0", because the field is a quantity and is not a pointer, but it's not defaulted per se, so they end up handling it as "1" in code: https://github.com/kubernetes/kubernetes/blob/34b85c593d13168edee426c63e97dbfbc760395b/pkg/api/v1/resource/helpers.go#L345C1-L347C10

I'd say that although we are currently implicitly setting it to 0, we can set it to 1 to be more explicit.

Reply:
Updated accordingly